Src/EB/AMReX_EB_triGeomOps_K.H

```diff
@@ -0,0 +1,303 @@
+#ifndef AMREX_EB_TRIGEOMOPS_K_H_
+#define AMREX_EB_TRIGEOMOPS_K_H_
+
+#include <AMReX.H>
+
+using namespace amrex;
+
+namespace tri_geom_ops
+{
+    //================================================================================
+    AMREX_GPU_HOST_DEVICE AMREX_INLINE Real Distance2(Real P1[3],Real P2[3])
```

Comment:
Can we use `AMREX_FORCE_INLINE` instead of `AMREX_INLINE`?

Reply:
using FORCE_INLINE for all functions now